entrypoint.sh

```diff
@@ -63,6 +63,22 @@ else
 			echo "Package HASH check failed"
 			exit 1
 		fi
+
+		PATCHES_DIR=`find feeds -path "*/$PKG/patches"`
```

Comment:
shellcheck will error on this, asking to replace with $(). I don't know if that will break it though.

Reply:
you mean the backticks? Can use $()